bin/clover/src/pipelines/gcp/funcs/actions/create.ts

```diff
@@ -1,11 +1,130 @@
+  if (!codeString) {
+    return {
+      status: "error",
+      message: "Could not find GCP Create Code Gen code for resource",
+    };
+  }
+
+  // Get method metadata from domain.extra
+  const insertMethodJson = _.get(
+    component.properties,
+    ["domain", "extra", "insertMethod"],
+    "",
+  );
+
+  if (!insertMethodJson) {
+    return {
+      status: "error",
+      message: "No insert method metadata found - this resource may not support creation",
+    };
+  }
+
+  const insertMethod = JSON.parse(insertMethodJson);
+  const baseUrl = _.get(component.properties, ["domain", "extra", "baseUrl"], "");
 
+  // Get authentication token
+  const serviceAccountJson = requestStorage.getEnv("GOOGLE_APPLICATION_CREDENTIALS_JSON");
   if (!serviceAccountJson) {
     throw new Error("Google Cloud Credential not found. Please ensure a Google Cloud Credential is attached to this component.");
   }
 
-  // Activate service account using stdin (most secure - no file on disk)
+  const { token, projectId } = await getAccessToken(serviceAccountJson);
+
+  // Build the URL by replacing path parameters
+  let url = `${baseUrl}${insertMethod.path}`;
+
+  // Replace path parameters with values from resource_value or domain
+  if (insertMethod.parameterOrder) {
+    for (const paramName of insertMethod.parameterOrder) {
+      let paramValue;
+
+      // Use extracted project_id for project parameter
+      if (paramName === "project") {
+        paramValue = projectId;
+      } else {
+        paramValue = _.get(component.properties, ["resource", "payload", paramName]) ||
+                     _.get(component.properties, ["domain", paramName]);
+      }
+
+      if (paramValue) {
+        url = url.replace(`{${paramName}}`, encodeURIComponent(paramValue));
+      }
+    }
+  }
+
+  // Make the API request
+  const response = await fetch(url, {
+    method: insertMethod.httpMethod,
+    headers: {
+      "Authorization": `Bearer ${token}`,
+      "Content-Type": "application/json",
+    },
+    body: codeString,
+  });
+
+  if (!response.ok) {
+    const errorText = await response.text();
+    return {
+      status: "error",
+      message: `Unable to create resource; API returned ${response.status} ${response.statusText}: ${errorText}`,
+    };
+  }
+
+  const responseJson = await response.json();
+
+  // Extract the primary identifier
+  let resourceId;
+
+  // GCP often returns an operation object for long-running operations
+  // Check if this is an operation response (has kind with "operation")
+  if (responseJson.kind && responseJson.kind.includes("operation") && responseJson.targetLink) {
+    // Extract resource name from targetLink (last segment of the path)
+    const targetLink = responseJson.targetLink;
+    const segments = targetLink.split("/");
+    resourceId = segments[segments.length - 1];
+  } else {
+    // Otherwise use name or id from the response
+    resourceId = responseJson.name || responseJson.id;
+  }
```

Comment:
If it is returning LRO links, we should poll those links until they complete (or timeout). We do this in azure and msgraph if you need examples. They are probably more complicated there than what you need here.